### PR TITLE
cephfs-top: run_display: clear the screen and block commands when no fs

### PR DESCRIPTION
**Pull request by joscollin**
This PR needs another round of testing.
FYI: This fix is not related to the scrolling feature. This fix is a continuation of https://github.com/ceph/ceph/commit/a386400c97868638eecdf0f8b20623cd97ec8791, which goes to the 'main' branch only. 

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [x] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by vshankar** [comment]
@joscollin Makes sense.

BTW, what happens when a file system is created in the interim?

**Comment by joscollin** [comment]
> @joscollin Makes sense.
> 
> BTW, what happens when a file system is created in the interim?

It generally shows up in cephfs-top and works as expected in `All Filesystem Info` screen. Will check how the `Selected Filesystem` screen behaves in such situation. That's why this PR is a draft.

**Comment by vshankar** [comment]
> It generally shows up in cephfs-top and works as expected in `All Filesystem Info` screen. Will check how the `Selected Filesystem` screen behaves in such situation. That's why this PR is a draft.

It would be ideal to automaticall unblock when a file system gets created.

**Comment by joscollin** [comment]
> > It generally shows up in cephfs-top and works as expected in `All Filesystem Info` screen. Will check how the `Selected Filesystem` screen behaves in such situation. That's why this PR is a draft.
> 
> It would be ideal to automaticall unblock when a file system gets created.

Yes, the fix is done and under testing. Will push after testing.

**Comment by joscollin** [comment]
@vshankar This is ready for review. It blocks the commands when there's no fs in the `Selected Filesystem Info` screen. It unblocks when the FS is created again. The user has the potion to quit anytime too.

**Comment by vshankar** [comment]
@neesingh-rh PTAL

File: src/tools/cephfs/top/cephfs-top [hunk 1]
```diff
@@ -843,19 +865,21 @@ class FSTop(object):
...
+            if cmd in [ord('m'), ord('s'), ord('l'), ord('r'), ord('q')]:
+                if self.set_option_sel_fs(cmd, fs):
+                    self.exit_ev.set()
+
             stats_json = self.perf_stats_query()
             vscrollEnd = 0
             if fs not in fs_list:
-                help = "Error: The selected filesystem is not available now. " + help_commands
+                help = f"Error: The selected filesystem '{fs}' is not available now. " \
+                    "[Press 'q' to quit]"
```
Comment (neesingh-rh) [inline src/tools/cephfs/top/cephfs-top:880]: It should be `Press 'q' to go back to the previous screen`. otherwise LGTM

Comment (joscollin) [reply src/tools/cephfs/top/cephfs-top:880]: thanks @neesingh-rh , will update the PR

**Review by neesingh-rh — APPROVED** [review]

**Pull request merged**